roles/passenger/molecule/default/converge.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: Converge
+  hosts: all
+  vars_files:
+    - ../../defaults/main.yml
+  vars:
+    - running_on_server: false
+    - app_name: molecule_test
+    - passenger_startup_file: 'test_wsgi.py'
+  become: true
+  pre_tasks:
+    - name: create minimal app dir
+      file:
+        dest: "{{ passenger_app_root }}"
+        state: directory
+        mode: 0655
+    - name: create minimal wsgi file
+      copy:
+        src: "test_wsgi.py"
+        dest: "{{ passenger_app_root }}/{{ passenger_startup_file }}"
+        mode: 0644
```

Comment:
could these be `prepare.yml` instead of `pre_tasks` here? that seems to be what it's for

Reply:
ah, sure. still learning the conventions!

Reply:
no worries, me too!